portfolio-alex.css

```diff
@@ -129,30 +216,65 @@ a{
 	text-align:center;
 	line-height:150px;
 	border-right: 1px solid #000000;
+	font-size:1.7rem;
 }
+@media only screen and (max-width: 900px) {
+  .contactholder h2{
+	  border-right:0;
+	  
+  }
+} 
+@media only screen and (max-width: 900px) {
+  .contactholder h2{
+	  width:100%;
+  }
+} 
```

Comment:
You have two media queries with the same characteristics and selectors, you should combine these.